<launch>
        <node name="human_pose_tf_broadcaster" pkg="lightweight_human_pose_estimation" type="human_pose_tf_broadcaster" output="screen">
            <!-- <param name="base_frame_name" value="base_link"/> -->
            <param name="base_frame_name" value="azure_camera_base"/>
            <param name="cloud_topic_name" value="/points2"/>
        </node>
</launch>